src/ai/backend/manager/models/base.py

```diff
@@ -1355,6 +1356,10 @@ async def populate_fixture(
         async with engine.begin() as conn:
             # Apply typedecorator manually for required columns
             for col in table.columns:
+                if isinstance(col.type, sa.sql.sqltypes.DateTime):
+                    for row in rows:
+                        if col.name in row:
+                            row[col.name] = isoparse(row[col.name])
```

Comment:
Wasn't there a datetime value acceptor before? Why did you add it?

Reply:
In the previous code for `populate_fixture`, the logic for handling `DateTime` types was missing, so an error occurs when generating a fixture that includes `DateTime` columns like `ImageRow.created_at`.